recipe/patch_yaml/nvcc.yaml

```diff
@@ -0,0 +1,19 @@
+# Fixes https://github.com/conda-forge/nvcc-feedstock/issues/108 for older builds by
+# appying https://github.com/conda-forge/nvcc-feedstock/pull/109/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a
```

Comment:
Looks like pre-commit doesn't like the line length. Perhaps we can just link the PR?

```suggestion
# applying https://github.com/conda-forge/nvcc-feedstock/pull/109
```

Another option may be linking the commit directly

```suggestion
# applying https://github.com/conda-forge/nvcc-feedstock/pull/109/commits/fea5e13ce4c72d1ddcde4adc6c2bac48b5a52447
```

Reply:
Thanks! I had already fixed that, just forgot to push from local.